src/widgets/hits-per-page/__tests__/hits-per-page-test.js

```diff
@@ -74,9 +77,13 @@ describe('hitsPerPage()', () => {
       ],
     });
 
-    expect(widgetWithDefaults.getConfiguration()).toEqual({
-      hitsPerPage: 20,
-    });
+    expect(
+      widgetWithDefaults.getConfiguration(new SearchParameters({}))
+    ).toEqual(
+      expect.objectContaining({
```

Comment:
I wonder if we should create `toEqualSearchParameters` or use `toEqual(new SearchParameters({`, I'm fine with `objectContaining` too since we aren't yet consistent about this

Reply:
I replaced this with `new SearchParameters({ ... })` to check the returned type as well.